2022-03-23 16:09:10,978 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:09:10,978 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:14:31,097 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:14:31,097 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:19:12,947 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:19:12,947 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:19:37,940 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:19:37,940 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:19:37,941 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:19:37,941 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:19:37,942 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:19:37,942 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:19:37,956 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd420860280>
2022-03-23 16:19:37,956 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd420860280>
2022-03-23 16:19:37,956 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd420860280>
2022-03-23 16:19:37,956 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd420860280>
2022-03-23 16:19:37,957 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd420860280>
2022-03-23 16:19:37,957 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd420860280>
2022-03-23 16:19:37,958 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd420860280>
2022-03-23 16:19:37,958 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd420860280>
2022-03-23 16:19:37,962 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:19:37,962 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:19:56,613 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:19:56,613 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:20:18,486 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:20:18,486 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:21:07,777 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:21:07,777 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:21:07,778 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:21:07,778 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:21:07,779 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:21:07,779 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:21:07,795 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fa3f5283d30>
2022-03-23 16:21:07,795 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fa3f5283d30>
2022-03-23 16:21:07,796 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fa3f5283d30>
2022-03-23 16:21:07,796 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fa3f5283d30>
2022-03-23 16:21:07,797 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fa3f5283d30>
2022-03-23 16:21:07,797 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fa3f5283d30>
2022-03-23 16:21:07,798 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fa3f5283d30>
2022-03-23 16:21:07,798 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fa3f5283d30>
2022-03-23 16:25:44,529 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:25:44,529 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:25:44,531 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:25:44,531 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:25:44,531 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:25:44,531 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:25:44,545 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd2991c48b0>
2022-03-23 16:25:44,545 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd2991c48b0>
2022-03-23 16:25:44,546 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd2991c48b0>
2022-03-23 16:25:44,546 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd2991c48b0>
2022-03-23 16:25:44,547 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd2991c48b0>
2022-03-23 16:25:44,547 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd2991c48b0>
2022-03-23 16:25:44,548 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd2991c48b0>
2022-03-23 16:25:44,548 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fd2991c48b0>
2022-03-23 16:29:24,216 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:29:24,216 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:29:24,217 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:29:24,217 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:29:24,218 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:29:24,218 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:29:24,233 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fe86d1c4820>
2022-03-23 16:29:24,233 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fe86d1c4820>
2022-03-23 16:29:24,234 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fe86d1c4820>
2022-03-23 16:29:24,234 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fe86d1c4820>
2022-03-23 16:29:24,235 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fe86d1c4820>
2022-03-23 16:29:24,235 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fe86d1c4820>
2022-03-23 16:29:24,236 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fe86d1c4820>
2022-03-23 16:29:24,236 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7fe86d1c4820>
2022-03-23 16:30:49,469 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:30:49,469 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:30:49,470 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:30:49,470 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:30:49,471 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:30:49,471 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:30:49,486 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f9d2cac4820>
2022-03-23 16:30:49,486 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f9d2cac4820>
2022-03-23 16:30:49,488 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f9d2cac4820>
2022-03-23 16:30:49,488 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f9d2cac4820>
2022-03-23 16:30:49,489 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f9d2cac4820>
2022-03-23 16:30:49,489 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f9d2cac4820>
2022-03-23 16:30:49,489 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f9d2cac4820>
2022-03-23 16:30:49,489 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f9d2cac4820>
2022-03-23 16:36:28,178 users.py:92  INFO search_user DoesNotExist, User notthere not found
2022-03-23 16:36:28,178 main.py:220 INFO add picture failed, notthere user not found.
2022-03-23 16:36:28,271 user_picture.py:47  INFO search_user_picture DoesNotExist, User notthere not found
2022-03-23 16:37:47,984 users.py:92  INFO search_user DoesNotExist, User notthere not found
2022-03-23 16:37:47,984 main.py:220 INFO add picture failed, notthere user not found.
2022-03-23 16:37:48,074 user_picture.py:47  INFO search_user_picture DoesNotExist, User notthere not found
2022-03-23 16:44:52,577 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:44:52,577 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:45:23,628 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:45:23,628 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:45:44,830 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:45:44,830 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:47:16,004 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:47:16,004 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:48:23,174 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:48:23,174 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:48:23,182 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:48:23,182 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:50:39,803 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:50:39,803 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:51:18,077 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:51:18,077 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:54:50,895 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:54:50,895 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 16:54:50,896 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:54:50,896 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 16:54:50,897 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:54:50,897 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 16:54:50,912 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f7f5205e280>
2022-03-23 16:54:50,912 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f7f5205e280>
2022-03-23 16:54:50,913 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f7f5205e280>
2022-03-23 16:54:50,913 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f7f5205e280>
2022-03-23 16:54:50,914 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f7f5205e280>
2022-03-23 16:54:50,914 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f7f5205e280>
2022-03-23 16:54:50,915 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f7f5205e280>
2022-03-23 16:54:50,915 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f7f5205e280>
2022-03-23 16:54:50,918 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:54:50,918 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:56:11,263 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:56:11,263 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:56:11,270 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:56:11,270 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 16:56:41,047 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:56:41,047 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:59:04,741 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 16:59:04,741 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 17:01:22,596 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:01:22,596 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:01:22,603 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:01:22,603 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:02:47,571 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:02:47,571 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:02:47,578 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:02:47,578 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:05:54,568 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 17:05:54,568 _internal.py:225 INFO  * Running on http://127.0.0.1:5002/ (Press CTRL+C to quit)
2022-03-23 17:08:43,854 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:08:43,854 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:08:43,861 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:08:43,861 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:09:19,998 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:09:19,998 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:09:20,006 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:09:20,006 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:12:45,303 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 17:12:45,303 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 17:12:45,303 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 17:12:45,303 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 17:12:45,304 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 17:12:45,304 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 17:12:45,320 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,320 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,321 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,321 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,322 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,322 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,323 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,323 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,326 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 17:12:45,326 users.py:45  INFO User cbarker12 not added, duplicate user found
2022-03-23 17:12:45,327 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 17:12:45,327 users.py:45  INFO User fjones34 not added, duplicate user found
2022-03-23 17:12:45,328 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 17:12:45,328 users.py:45  INFO User bwinkle678 not added, duplicate user found
2022-03-23 17:12:45,344 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,344 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,345 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,345 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,346 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,346 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,347 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:12:45,347 logging_setup_function.py:37  ERROR Exception: UNIQUE constraint failed: UserStatusTable.STATUS_ID in <function UserStatusCollection.add_status at 0x7f8eabae48b0>
2022-03-23 17:15:44,651 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:15:44,651 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:15:44,659 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:15:44,659 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:20:10,824 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:20:10,824 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:20:10,876 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:20:10,876 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:22:27,687 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:22:27,687 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:22:51,623 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:22:51,623 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:23:21,541 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:23:21,541 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:23:27,241 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
2022-03-23 17:23:27,241 user_picture.py:47  INFO search_user_picture DoesNotExist, User None not found
